### ISSUE TITLE
ValueError: Unrecognized configuration class <class 'transformers.models.t5.configuration_t5.T5Config'> for this kind of AutoModel: AutoModelForSequenceClassification.

### ISSUE THREAD

**Issue by harishprabhala** [ELS-RD/transformer-deploy#120]
Hi - I am trying to accelerate some T5 models and I get this error. How do I fix this?

Command to reproduce:

`convert_model -m "valhalla/t5-small-qa-qg-hl" --backend tensorrt onnx --seq-len 16 16 16`

I understand that this requires a AutoModelForSeq2SeqLM, or MT5ForConditionalGeneration class. How do I add these and how to add a new `--task seq2seq ` apart from the existing classes? I get this error when I add a new task seq2seq
`
-task: invalid choice: 'seq2seq' (choose from 'classification', 'embedding', 'text-generation')`

Thanks

**Comment by pommedeterresautee** [comment]
Convert command doesn't manage t5 for now, you may follow the notebook to get the trt model

**Comment by harishprabhala** [comment]
Any plans to add T5 support for the convert command in the near future?

**Comment by pommedeterresautee** [comment]
Yes the Triton server with onnx 1.12 (required for that) has been released few days ago and we were waiting for it. However I have no ETA to share for now.
